minecraft/1.7/src/main/java/nova/core/wrapper/mc/forge/v17/wrapper/item/OreDictionaryIntegration.java

```diff
@@ -44,43 +48,44 @@ private OreDictionaryIntegration() {
 	public void registerOreDictionary() {
 		ItemDictionary novaItemDictionary = Game.itemDictionary();
 
-		for (String oredictEntry : novaItemDictionary.keys()) {
-			for (Item oreValue : novaItemDictionary.get(oredictEntry)) {
-				OreDictionary.registerOre(oredictEntry, ItemConverter.instance().toNative(oreValue));
-			}
-		}
+		novaItemDictionary.stream().forEach(entry -> entry.getValue().stream()
```

Comment:
It's quite unclear that there are actually two nested `forEach`s here, I would suggest using a `-> {}` block to make it more explicit.